lib/util.js

```diff
@@ -93,6 +93,17 @@ const util = {
     fs.copySync(path.join(braveComponentsDir, 'resources', 'default_200_percent', 'brave'), path.join(chromeComponentsDir, 'resources', 'default_200_percent', 'chromium'))
     fs.copySync(path.join(braveAppVectorIconsDir, 'vector_icons', 'brave'), path.join(chromeAppDir, 'vector_icons', 'brave'))
     fs.copySync(path.join(braveResourcesDir, 'settings', 'brave_page_visibility.js'), path.join(chromeResourcesDir, 'settings', 'brave_page_visibility.js'))
+
+    if (process.platform === 'darwin') {
+      // Copy proper mac app icon for channel to chrome/app/theme/mac/app.icns.
+      // Each channel's app icons are stored in brave/app/theme/$channel/app.icns.
+      // With this copying, we don't need to modify chrome/BUILD.gn for this.
+      let channel = config.channel
+      if (config.debugBuild)
```

Comment:
shouldn't this already be correct in `config.channel`? If not I think we should set it in config.js and not here

Reply:
Ah,, debug build uses different app icon with other four channel app icons.

Reply:
understood, what I'm saying is that the channel name (development) should be set in config.js, not here

Reply:
> Ah, do you mean setting `development` to `config.channel` in debug build?

Oh, my previous comment was pending state :) I'll try to check this channel setting doesn't break anything.

Reply:
it should be checking the channel name here and doing any required icon name mapping, not checking debugBuild

Reply:
you don't necessarily have to set the channel to `development`, but it should be set to whatever makes sense for debugBuilds and then map from that to `development` for the icon path

Reply:
How about using `development` for channel in debug build?
I think it's fine because icon is stored under `development` folder/

Reply:
fine with me, but check with @mbacchi to be sure

Reply:
I think it would be fine because debug build is only used for local development build.
WDYT? @mbacchi

Reply:
Updated to set `development` to channel name.
How about merging this first and revisit here when @mbacchi  comments something because mac release build is broken for now?